content/beginner/session1.md

```diff
@@ -2,7 +2,7 @@
 title: "Introduction to Python"
 number: 1
 date: 2023-02-23T00:00:00-00:00
```

Comment:
and this should be the 16th, then once a week for each of the sessions after that